Makefile

```diff
@@ -9,4 +9,5 @@ test:
 	go test ./...
 
 build:
-	cd cmd && go build -o ../bin/relayer 
\ No newline at end of file
+	mkdir -p bin
+	GOBIN="$(CURDIR)/bin/" go install ./cmd/...
```

Comment:
why `go install` vs `go build`? don't need to change this, just curious about the reasoning/difference

Reply:
Once we put a release tag on the repo, you can use the tag as part of the `go install` to install a specific release. For this line specifically, we are installing off of the latest commit, not a tag.  If this latest commit is tagged, the version tag should show up in our version sting regardless. We can verify that when we do our first tag.